src/index.html

```diff
@@ -9,10 +9,72 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
Ensure that the BEM naming convention is followed consistently. If `calendar--start-day-sun` and `calendar--month-length-31` are modifiers, they should be used with the base class `calendar` on the same element, e.g., `class="calendar calendar--start-day-sun calendar--month-length-31"`.